login/service.go

```diff
@@ -734,7 +682,8 @@ func (keycloak *KeycloakOAuthProvider) CreateOrUpdateKeycloakUser(accessToken st
 			return nil, nil, errors.New("Cant' create user/identity " + err.Error())
 		}
 	} else {
-		user = &identities[0].User
+		identity = identities[0]
```

Comment:
We need to set both user and identity (we return them). So should be separate operations.